tests/ImageStreamIO_subTest_Operations.cpp

```diff
@@ -54,6 +54,24 @@ install_sigchld_handler()
     return 0;
 }
 
+#   define WAIT_AS_ns 100000000
+#   define WAIT_LIMIT_ns (1000000000 - WAIT_AS_ns)
+
+int
+clock_gettime_future(struct timespec& ts)
+{
+    if (clock_gettime(CLOCK_REALTIME,&ts)) { return -1; }
+    if (ts.tv_nsec < WAIT_LIMIT_ns) {
+        ts.tv_nsec += WAIT_AS_ns;
+    }
+    else
+    {
+        ts.tv_nsec -= WAIT_LIMIT_ns;
+        ++ts.tv_sec;
+    }
+    return 0;
+}
+
```

Comment:
Utility for operational test that follows; this routine calculates a time 100,000,000ns (0.1s) in the future